vllm/model_executor/layers/triton_kernel/prefix_prefill.py

```diff
@@ -5,6 +5,8 @@
 import triton
 import triton.language as tl
 
+TESLA = 'Tesla' in torch.cuda.get_device_name(0)
```

Comment:
Can we set this variable in a function instead of a global variable? Setting it in global variable may lead to issues in distributed setting.